exercises/sum.cpp

```diff
@@ -6,3 +6,20 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <iostream>
+
+using namespace std;
+
+int main(int argc, char** argv){
+  
+  int n1,n2;
+
+  cout<<"Insert the first number: ";
+  cin>>n1;
+  cout<<endl<<"Insert the second number: ";
+  cin>>n2;
+  cout<<endl<<"Sum: "<<n1+n2;
+
+  return 0;
```

Comment:
Il `return 0` è poco esplicativo di per sè.
Potresti fare un:
`#define EXIT_SUCCESS 0` e inserire alla fine del main `return EXIT_SUCCESS;` in modo tale da incrementare la leggibilità del codice.